### ISSUE TITLE
How to extract unsupervised loss value from training?

### ISSUE THREAD

**Issue by wanglu2014** [eliorc/node2vec#69]
Since loss is an important measure to optimize the model, how can we extract loss from the model?

**Comment by eliorc** [comment]
The `.fit`method as stated in the docs can accept any argument that `gensim.Word2Vec` accepts, so following the documentation of gensim, you can just do the following

```python3
node2vec = Node2Vec(graph, ...)

# Add compute_loss=True
model = node2vec.fit(..., compute_loss=True)

# Get the loss
loss = model.get_latest_training_loss()
print(loss)
```

**Comment by wanglu2014** [comment]
Thank you for your reply!
